### ISSUE TITLE
Feature/configuration request/question: Spacing between `Image` and its `caption`.

### ISSUE THREAD

**Issue by rydmike** [invertase/docs.page#173]
## Configure Image Caption Spacing

Is it possible to configure the spacing between `Image` and its `caption`?
If so, how? If not, is it possible to introduce the capability?

The spacing is very large now, I would prefer it to be much smaller.

![image](https://user-images.githubusercontent.com/39990307/163256672-290719d1-b580-4a6f-9326-533d4e107eff.png)

### Reproduction docs repo:

Example can be seen here: https://docs.page/rydmike/docs_page_test/another_page

Here is a test reproduction repo: https://github.com/rydmike/docs_page_test
and site: https://docs.page/rydmike/docs_page_test

Having `docs.json` config file:
```json
{
    "name": "DocsPageTests",
    "favicon": "/images/favicon.png",
    "logo": "/images/logo.png",
    "logoDark": "/images/logo.png",
    "theme": "#0x29B6F6",
    "zoomImages": "true",
    "noindex": "false",
    "sidebar": [
        ["Welcome",
            [
                ["Intro","/"],
                ["More intro", "/more_intro"]
            ]
        ],
        ["Zoom image", "/another_page"]
    ],
    "navigation": [
        ["Intro","/"],
        ["More intro", "/more_intro"],
        ["Package", "https://pub.dev/packages/flex_color_scheme"],
        ["Playground", "https://rydmike.com/flexcolorscheme/themesplayground-v5"]
    ]
}
```

**Comment by cabljac** [comment]
Hey, when i fixed zoom I believe i adjusted this slightly, can you let me know if it's sufficient? If not then i'm happy to decrease it further or even set it as customisable

**Comment by rydmike** [comment]
Hi Jacob, yes I noticed also already earlier, about after that image fix. That makes sense now why I did not think it was as large as before when I saw it. 

Definitely better now, still a bit large for my personal preference, I would probably use only about half of what it still is. That's me coming from writing scientific publications where an image caption is usually quite tight with the image. Styles and preferences do vary though, so who is to say what is right and wrong, only a preferences. So this is much better imo, if it were configurable, I would as mention go with even tighter, but if this is what I get, I will take it too! 😄 

Having more and easily configurable parameters, will of course make the docs.page more attractive in the long run when more are using that may prefer different styles and settings.

In any case much better already - thanks!

![image](https://user-images.githubusercontent.com/39990307/165740990-0a6c6bde-0ae6-40b7-afda-9751ad413c26.png)
